src/providers/staticfile.rs

```diff
@@ -116,3 +124,21 @@ impl Provider for StaticfileProvider {
         ))))
     }
 }
+
+impl StaticfileProvider {
+    pub fn get_root(app: &App, env: &Environment, staticfile_root: String) -> String {
+        let mut root = "";
+        if let Some(staticfile_root) = env.get_variable("NIXPACKS_STATICFILE_ROOT") {
```

Comment:
nit/for future: Can use `get_config_variable` that automatically prefixes with "NIXPACKS_"

Reply:
sure, lemme make that change.

Reply:
Changing how that function works should be in a different PR to update all the current uses of. NIXPACKS_ variables.